spec/models/rent_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe Rent, type: :model do
+  it { validate_presence_of(:book_id) }
+  it { validate_presence_of(:user_id) }
+  it { validate_presence_of(:from) }
+  it { validate_presence_of(:to) }
+
+  let(:user) { create(:user) }
+  let(:book) { create(:book) }
+
+  subject(:rent) do
```

Comment:
Use curly brackets if the block it's shorther than one line.

`subject(:rent) { create(:rent, user: user, book: book) }`